pages/index.js

```diff
@@ -68,7 +68,7 @@ export default function Index() {
         { filterResults(offers[category], searchQuery).length === 0 ?
           <span>No Results</span>
           : filterResults(offers[category], searchQuery).map(offer => {
-          return <li className="offer">
+          return <li className="offer" key={JSON.stringify(offer)}>
```

Comment:
`JSON.stringify(offer)` is quite a big key... Anything smaller that can be used?

Reply:
yeah this stuff is f annoying, theres another one introduced in master